### ISSUE TITLE
[Enhancement] contrast-colors for kirby-item

### ISSUE THREAD

**Issue by MonsterGameDev** [kirbydesign/designsystem#2012]
## Describe the enhancement
The kirby-item component should change font-colors etc when background-color changes (fx. light to dark)

## Describe the solution you'd like
I would like to be able to read text, see buttons etc in kirby-item when placed on dark backgroundm like tertiary

## Have you considered any alternatives?
The kirby item uses projection, so you can actually control almost all styling when using a kirby item. I my case though i did not directly implement the kirby-item, but rather the shared TaxConsentComponent which itself implements the kirby-item... And - due to encapsulation - i would have to use ngDeep or css-variables to access the styling.

## Are there any additional context?
One could say: I beleive all kirby-components should react this way

This issue has been discussed and screenshots/codeexamples are available in Kirby Slack-channel during open hours: https://jyskebank-workspace.slack.com/archives/CGKJ1RBM4/p1643116017039200

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
